### PR TITLE
Added Plugin Directory Submission Tips

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added Plugin Directory Submission Tips and also add the same for when doing reviews

This should help with https://github.com/backstage/backstage/issues/21513 but is not a perfect solution

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/plugins/add-to-directory.md [hunk 1]
```diff
@@ -25,3 +25,14 @@ iconUrl: # Used as the src attribute for your logo.
 npmPackageName: # Your npm package name E.g. '@backstage/plugin-<etc>' quotes are required
 addedDate: # The date plugin added to directory E.g. '2022-10-01' quotes are required
 ```
+
+## Submission Tips
+
+Here are a few tips to help speed up the review process when you submit your plugin:
+
+- For any icon that you use make sure you have the proper rights to use it.
```
Comment (tudi2d) [inline docs/plugins/add-to-directory.md:33]: on the icon: I believe the example for the `iconUrl` above is misleading. When using a relative path to static the `iconUrl` should start with a `/`, right?

**Review by tudi2d — APPROVED** [review]
thanks for the addition to the documentation

File: docs/plugins/add-to-directory.md (hunk 1 above)
Comment (awanlin) [reply docs/plugins/add-to-directory.md:33]: Good call, this actually just came up: https://github.com/backstage/backstage/pull/21524#discussion_r1404407622

I've added the `/` 👍

**Comment by benjdlambert** [comment]
Thanks :tada:

**Pull request merged**